addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -71,6 +71,11 @@ $spacers: (
 $body-bg: o-color('o-cc1-bg') !default;
 $body-color: o-color('o-cc1-text') or color-contrast($body-bg) !default;
 
+// Bootstrap 5.3 introduced the tertiary-bg variable which is now the default value on hover of the
+// following components: dropdown-item, pagination-item, list-group-item. To ensure
+// contrast with the user's palette, a relative value of the bg-color is used.
```

Comment:
80 chars max per line for comments please :pray: Also detail: maybe do not mention all components (say something like "e.g. dropdown-item", otherwise this has a huge chance to be wrong in a few weeks because we'll have changed something :shrug:

Reply:
@mano-odoo Also, this PR should actually target 17.4 and not 18.0 please :pray: